WalletWasabi.Fluent/Controls/CurrencyEntryBox.axaml.cs

```diff
@@ -128,6 +128,12 @@ protected override void OnGotFocus(GotFocusEventArgs e)
 
 	protected override void OnTextInput(TextInputEventArgs e)
 	{
+		if (Text.Trim() == "" && e.Text == ".")
+		{
+			InsertImplicitZeroForDecimal(e);
+			return;
+		}
+
```

Comment:
Shouldn't this be after the sanity check? and use `input` instead of `e.Text` for consistency.

Reply:
Fixed :)